core/store/src/db.rs

```diff
@@ -485,7 +485,10 @@ fn rocksdb_options() -> Options {
 fn rocksdb_block_based_options() -> BlockBasedOptions {
     let mut block_opts = BlockBasedOptions::default();
     block_opts.set_block_size(1024 * 16);
+    #[cfg(feature = "no_cache")]
+    let cache_size = 1024 * 1024 * 16;
```

Comment:
should we set it to an even smaller number? like 2MB?

Reply:
I tried setting it to 1mb before, but it affected performance too much, the node couldn't run anymore.
I have another PR, which add performance metrics. We should first measure how those settings affect the performance, before we do any changes.

That's why I don't want to push this PR, before I get the data.

Reply:
I see. Makes sense